src/early-types.lisp

```diff
@@ -312,7 +312,7 @@ Signals an error if FOREIGN-TYPE is undefined."))
   ())
 
 (defun follow-typedefs (type)
-  (if (typep type 'foreign-type-alias)
+  (if (typep type 'foreign-typedef)
```

Comment:
This breaks whatever f8f7ac883a708febf1032574f519e36e27bbe6d2 was trying to fix, so I've added a note to https://bugs.launchpad.net/cffi/+bug/1474631 about it.